hw/ip/dma/data/dma.hjson

```diff
@@ -355,7 +359,7 @@
       ]
     }
     { name: "CHUNK_DATA_SIZE"
-      desc: "Total size of the data blob involved in DMA movement."
+      desc: "Size of a single data blob involved in DMA movement."
```

Comment:
Suggestion: "Number of bytes to be transferred as a discrete unit in response to a single interrupt/firmware request."